### ISSUE TITLE
Jobs staying open after close date?

### ISSUE THREAD

**Issue by curtiskphsc** [phsc-edu/2018#399]
Pat reports this job https://phsc.edu/jobs/regular-positions/executive-administrative-assistant-institutional-technology-distance was scheduled to close on 8/24/2018 at 11:59 p.m. and it is still visible to anonymous users. See attached. Please review and/or close.
![anonymous-view-of-job-not-closed](https://user-images.githubusercontent.com/28155245/44728755-316a9c00-aaab-11e8-824b-09be258fc78b.PNG)
![jobs-not-closing-as-scheduled](https://user-images.githubusercontent.com/28155245/44728768-37f91380-aaab-11e8-9944-1b5bf9d923cd.PNG)

**Comment by C13L0** [comment]
@jeppy64 Can you please take a look at this? Thanks!

**Comment by jeppy64** [comment]
@C13L0  @curtiskphsc 

We had added a closing date/time field for information display only. We also had node scheduler installed but once we enabled moderation on jobs, the scheduler no longer would work because the content was published under moderation.

We adjusted the existing view to filter on jobs with the new closing date, another display for jobs with "open unitl filled" and a third for all other - adjunct temp

This should take care of all scenarios.

Time to complete: 1/2 hour
